### PR TITLE
chore(tests): split test scripts and seperate bff-sdk-generation from…

### PR DESCRIPTION
**Pull request by bjwswang**
… example-test

#### Which issue(s) this PR fixes

Fixes #

### CODE REVIEW

File: .github/workflows/bff-sdk.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+name: KubeagiTests
```
Comment (Abirdcfly) [inline .github/workflows/bff-sdk.yaml:1]: change it?

File: apiserver/graph/schema/application.gql [hunk 2]
```diff
@@ -1,3 +1,4 @@
+# commoen for test
```
Comment (Abirdcfly) [inline apiserver/graph/schema/application.gql:1]: remove this line?

Comment (bjwswang) [reply apiserver/graph/schema/application.gql]: to trigger the gql test

Comment (bjwswang) [reply apiserver/graph/schema/application.gql]: to trigger the gql test

**Review by Abirdcfly — APPROVED** [review]
LGTM

**Pull request merged**